certify-csvdp-farmer.properties

```diff
@@ -0,0 +1,109 @@
+## ------------------------------------------- Mock ID Integration properties ------------------------------------------------------------
```

Comment:
this is a usecase specific file, do we need this in the release branch ?

Reply:
I believe we should have only the 4 plugin property files.

certify-mosipid-identity.properties - Uses MOSIP identity VC Issuance plugin and showcases identity usecase
certify-sunbird-insurance.properties - Uses Sunbird RC VC Issuance plugin and showcases insurance usecase
certify-mock-identity.properties - Uses mock data provider plugin and showcases identity usecase
certify-postgres-farmerid.properties / certify-postgres-landregistry.properties - Uses postgres data provider plugin and showcases farmerId/landregistry usecase

We have to update the below section also based on the above decisions. and remove the other unwanted files
https://github.com/mosip/inji-config/tree/develop?tab=readme-ov-file#inji-certify

Reply:
Removed usecase specific files and updated thee read.me with above specified plugins.